src/datadog/tracer_config.cpp

```diff
@@ -360,6 +360,13 @@ Expected<FinalizedTracerConfig> finalize_config(const TracerConfig &config) {
   result.report_hostname = config.report_hostname;
   result.tags_header_size = config.tags_header_size;
 
+  if (auto enabled_env =
+          lookup(environment::DD_TRACE_128_BIT_TRACEID_ENABLED)) {
+    result.trace_id_128_bit = !falsy(*enabled_env);
```

Comment:
Yes, that's the desired behavior.

As we discussed in our standup yesterday, the falsiness behavior of this library could use revisiting. For now, though, I think that
```
export DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED=''
```
should enable the feature, if only because of the word "enabled" in there. Reminds me of the tension in C preprocess macros between `#ifdef` and `#if`.